sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/LookupCatalog.scala

```diff
@@ -179,7 +176,7 @@ private[sql] trait LookupCatalog extends Logging {
           }
         }
 
-      case _ => throw new AnalysisException(s"$sql is only supported in v1 catalog")
+      case _ => throw new AnalysisException("function is only supported in v1 catalog")
```

Comment:
function -> function command?

Reply:
This is called from `Analyzer.scala` except for `CREATE FUNCTION`, so I thought more general name would be better here. But if you think `function command` is better here, I am happy to change it. Please let me know!